docs/build.rst

```diff
@@ -65,7 +67,7 @@ variants.
                (``spack install <package>``)
    ==========  =============================  =======  ===========================
    Auto-mount  ``unifyfs+auto-mount``         True     Enable transparent mounting
-   Boostsys    ``unifyfs+boostsys`            False    Have Mercury use Boost
+   Boostsys    ``unifyfs+boostsys``           False    Have Mercury use Boost
    Fortran     ``unifyfs+fortran``            True     Enable Fortran support
    PMI         ``unifyfs+pmi``                False    Enable PMI2 support
    PMIx        ``unifyfs+pmix``               False    Enable PMIx support
```

Comment:
Should we also go ahead and document the `+preload` variant?

Reply:
Ah, yep, good catch.

Reply:
It's been added.